var/spack/repos/builtin/packages/mysql/package.py

```diff
@@ -226,3 +226,12 @@ def setup_build_environment(self, env):
 
         if "python" in self.spec and self.spec.satisfies("@:7"):
             self._fix_dtrace_shebang(env)
+
+    @run_before("install")
```

Comment:
Is this eternally valid? Is it not needed anymore starting with a specific version of mysql? Essentially, when can we get rid of this code?

Reply:
Good question! I didn't check newer versions yet, but I can re-test when you added the new versions!

As this just does filter_file in `-lzlib` if the files exist, it should be harmless even when it is not needed.

But feel free to go any direction you think is best!

If you like to remove these changes and merge with this commit (due to the upcoming MySQL PR) it's fine too!

Reply:
I'm marking this as resolved, and we can come back to it when upgrading the mysql package.